internal/pkg/cli/job_deploy.go

```diff
@@ -41,6 +45,7 @@ type deployJobOpts struct {
 
 	store              store
 	ws                 wsJobDirReader
+	fs                 *afero.Afero
```

Comment:
just to make sure do we need to update anything in the `init.go` file since we touched the `deploy` commands?

Reply:
good call! Added to both `init.go` and `deploy.go`